spec/lib/jets/controller/callbacks_spec.rb

```diff
@@ -65,6 +65,15 @@ def first; end
   def second; end
 end
 
+class SkippedBeforeControllerWithOnly < Jets::Controller::Base
+  before_action :first
+  skip_before_action :first, only: %i[second]
+
+  def first; end
+  def second; end
```

Comment:
I think second should be renamed to e.g. action or index as this is what it is in this scenario. As I read the scenario, `first` is a before action which gets skipped for the actual action `second`. Using `second` in this case for the action makes it quite hard to understand.

Reply:
Thanks for the feedback, ill update this spec specifically, i used `first` and `second` since all other specs use those methods, ill only update my new specs to keep the changes as minimal in the PR.